source/openapi-admin-v3.yaml

```diff
@@ -7102,6 +7070,590 @@ components:
+                type: string
+                example: "63ee5785bbfa4b9457f8509e"
+                description: |
+                  The ID of the data source that the schema was generated from.
+      SchemaValidation:
+        allOf:
+          - $ref: "#/components/schemas/Logs/Parts/BaseLog"
+          - $ref: "#/components/schemas/Logs/Parts/ClientLog"
+          - $ref: "#/components/schemas/Logs/Parts/Messages"
+          - $ref: "#/components/schemas/Logs/Parts/RuleMetrics"
+          - title: "SCHEMA_VALIDATION"
+            properties:
+              type:
+                const: "SCHEMA_VALIDATION"
+              service_id:
+                type: string
+                example: "63ee5785bbfa4b9457f8509e"
+                description: |
+                  The ID of the data source that was validated against the schema.
+      LogForwarder:
+        allOf:
+          - $ref: "#/components/schemas/Logs/Parts/BaseLog"
+          - $ref: "#/components/schemas/Logs/Parts/FunctionCall"
+          - $ref: "#/components/schemas/Logs/Parts/Messages"
+          - $ref: "#/components/schemas/Logs/Parts/BilledMemory"
+          - title: "LOG_FORWARDER"
+            properties:
+              type:
+                const: "LOG_FORWARDER"
+              event_subscription_id:
+                type: string
+                example: "641caa28ce889d60d32b9fe2"
+                description: |
+                  The underlying log event subscription's unique objectId hex.
+              event_subscription_name:
+                type: string
+                example: "toMongo_event_subscription"
+                description: |
+                  The underlying log event subscription's name.
+              log_forwarder_id:
+                type: string
+                example: "641caa28ce889d60d32b9fe2"
+                description: |
+                  The log forwarder's unique objectId hex.
+              log_forwarder_name:
+                type: string
+                example: "toMongo"
+                description: |
+                  The log forwarder's name.
+              failed_to_forward_log_ids:
+                type: array
+                items:
+                  type: string
+                description: |
+                  An array of log ``_id`` values for any logs that were not successfully forwarded.
+              data_source_id:
+                type: string
+                example: "63ee5785bbfa4b9457f8509e"
+                description: |
+                  The ID of the forwarded log data source.
+              data_source_name:
+                type: string
+                example: "mongodb-atlas"
+                description: |
+                  The name of the forwarded log data source.
+              data_source_namespace:
+                type: string
+                example: "myApp.logs"
+                description: |
+                  The `database.collection` namespace of the forwarded log collection.
```

Comment:
should this be 
```suggestion
                  The ``database.collection`` namespace of the forwarded log collection.
```

Reply:
The OpenAPI spec uses markdown, not reStructuredText, so the single backticks are actually correct here